openml_OS/views/pages/api_new/v1/xml/data-get.tpl.php

```diff
@@ -3,6 +3,7 @@
   <oml:name><?php echo htmlspecialchars($name); ?></oml:name>
   <oml:version><?php echo htmlspecialchars($version); ?></oml:version>
   <oml:description><?php echo htmlspecialchars($description); ?></oml:description>
+  <oml:description_version><?php echo htmlspecialchars($description_version); ?></oml:description_version>
```

Comment:
OK, but if you add this you also need to update the xsd file: https://github.com/openml/OpenML/blob/develop/openml_OS/views/pages/api_new/v1/xsd/openml.data.upload.xsd